### PR TITLE
owen - calorielist

### LINKED ISSUE #14: Implement data structure for date and activities

### LINKED ISSUE #15: As a user I want to list activities for the day

### PR DESCRIPTION
**Pull request by chewyang**

### CODE REVIEW

File: src/main/java/seedu/duke/ActivityMap.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+        if (alist == null) { //if the day specified does not contain an activity list yet, create one first
+            activitiesMap.put(dateTime.toLocalDate(), new ActivityList());
+        }
+        alist = this.getValue(dateTime);
+        alist.addActivity(activity);
+    }
+
+    //returns the activityList for the given dateTime
+    public ActivityList getValue(LocalDateTime dateTime) {
```
Comment (1-Karthigeyan-1) [inline src/main/java/seedu/duke/ActivityMap.java:34]: Could be more specific in naming convention. For example, "getList" or "getActivityList"

**Review by 1-Karthigeyan-1 — COMMENTED** [review]
Overall, looks good

File: src/main/java/seedu/duke/ActivityList.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ * List of activities for any day.
+ */
+public class ActivityList {
+    //Integer to be replaced with the Activity class
+    private ArrayList<Integer> activities;
+    private int activityCounter;
+
+    public ActivityList() {
+        activities = new ArrayList<>();
+        activityCounter = 0;
```
Comment (siewwencode) [inline src/main/java/seedu/duke/ActivityList.java:17]: Could have a final static for this magic number

File: src/main/java/seedu/duke/ActivityMap.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    private HashMap<LocalDate, ActivityList> activitiesMap;
+
+    public ActivityMap() {
+        this.activitiesMap = new HashMap<>();
+    }
+
+    //extracts activity list from map and adds the activity to it
+    //Integer to be replaced with activity class added this just to test this implementation
+    public void addActivity(LocalDateTime dateTime, Integer activity) {
+        ActivityList alist = this.getValue(dateTime);
```
Comment (siewwencode) [inline src/main/java/seedu/duke/ActivityMap.java:24]: alist should be in camelCase

File: src/main/java/seedu/duke/Duke.java [hunk 4]
```diff
@@ -17,5 +20,18 @@ public static void main(String[] args) {
 
         Scanner in = new Scanner(System.in);
         System.out.println("Hello " + in.nextLine());
+
+        //example code to use calorie list
+        ActivityMap calList = new ActivityMap();
+
+        LocalDateTime adatetime = LocalDateTime.of(2015,
```
Comment (siewwencode) [inline src/main/java/seedu/duke/Duke.java:27]: adatetime should be in camelCase

**Pull request merged**